scripts/run-e2e-tests.sh

```diff
@@ -69,24 +124,32 @@ function install_cluster() {
 
   echo "step 2/6: installing helm-v3"
   check_cmd "curl"
-  curl -L https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash
-  exit_on_error "install helm-v3 failed"
-  check_cmd "helm"
+  if ! command -v helm &> /dev/null
+  then
+      install_helm
+  else
+    echo "Helm already installed"
+  fi
```

Comment:
just a small suggestion, the check can be added in the `install_helm` function, which makes the main flow simpler.
same as the rest install functions.

Reply:
Good point. Looks neater now :)